### ISSUE TITLE
Open JRE for windows

### ISSUE THREAD

**Issue by Pconti31** [corretto/corretto-8#12]
Binary files for the OpenJDK 8, 10, and 11 are available from a number of sources.
There are a limited number of binary releases for the JRE and mostly for linux.
No 0ne supplies any JRE binaries for windows except Oracle.

It would be, I think, a nice feature enhancement for you to also provide the JRE's.  
The standard build creates the JRE images but no one provides or supports them as far as I know.

Just a thought.

**Comment by ziyiluo** [comment]
Thanks for the suggestion. May I ask about your use case? Are you looking for a headless JRE?

**Comment by Pconti31** [comment]
Yes, headless JRE.  The use case is that we have open source GUI for embedded systems.
Its both an API and GUI builder or Designer written in java.  We also have other apps in the works so
while I can use JLINK to create special JRE we would need one per app.  It would seem better to use
one installed JRE on windows.
You can see the project here:
https://github.com/ImpulseAdventure/GUIslice/wiki/GUIslice-Builder


[https://avatars3.githubusercontent.com/u/8510097?s=400&v=4]<https://github.com/ImpulseAdventure/GUIslice/wiki/GUIslice-Builder>

ImpulseAdventure/GUIslice<https://github.com/ImpulseAdventure/GUIslice/wiki/GUIslice-Builder>
GUIslice embedded touchscreen GUI library in C for Raspberry Pi, Arduino & ESP8266 / ESP32, utilizing SDL 1.2, SDL2, Adafruit-GFX or TFT_eSPI touch TFT - ImpulseAdventure/GUIslice
github.com



________________________________

**Comment by alvdavi** [comment]
We have now updated those zip files to include OpenJFX (marked as preview2_1)
